meson.build

```diff
@@ -34,11 +34,12 @@ config_piper.set('pkgdatadir', pkgdatadir)
 config_piper.set('localedir', localedir)
 config_piper.set('gtk_major_version', gtk_major_version)
 config_piper.set('gtk_minor_version', gtk_minor_version)
+config_piper.set('devel', '')
 
 configure_file(input: 'piper.in',
 	       output: 'piper',
 	       configuration: config_piper,
-	       install_dir: bindir )
+	       install_dir: bindir)
```

Comment:
for whitespace changes: please keep them in separate patches and push them directly.